projects/conding-challenge-i/src/App.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react'
+import './App.css'
+
+import CardName from './components/CardName'
+
+const randomInt = (min:number, max:number) => Math.floor(Math.random() * (max - min + 1)) + min;
+
+function findClosest(arr:number[],  target:number)
+{
+  let n = arr.length;
```

Comment:
ProTip: Intenta que las variables y constantes tengan nombres significativos, para que sea mas sencillo encontrar "La aguja en el pajar" cuando un sistema crece
```suggestion
  const n = arr.length;
```